Common/gfx/bitmap.h

```diff
@@ -77,6 +77,12 @@ namespace BitmapHelper
     // Pitch is given in bytes and defines the length of the source scan line.
     // Offset is optional and defines horizontal offset, in pixels.
     void    ReadPixelsFromMemory(Bitmap *dst, const uint8_t *src_buffer, const size_t src_pitch, const size_t src_px_offset = 0);
+
+    // Write to the stream a bitmap, optionally along with the palette
+    void SaveBitmapStream(Stream *out, const Bitmap *bmp, const RGB* pal = nullptr);

```

Comment:
I'd propose to call the functions simply LoadBitmap and SaveBitmap, having them differentiate with parameter types (if loading from file should still be kept).

Reply:
renamed!